jest.acceptance.config.js

```diff
@@ -1,4 +1,3 @@
-/* eslint-disable filenames/match-regex */
```

Comment:
perhaps the rule could allow any `*.config.js` files

Reply:
it does